scripts/docker-integration-tests/setup.sh

```diff
@@ -5,7 +5,7 @@ set -xe
 # expected to be run from root of repository
 cd $GOPATH/src/github.com/m3db/m3
 
-SERVICES=(m3dbnode m3coordinator m3aggregator)
+SERVICES=(  m3coordinator  )
```

Comment:
nit: Revert changes to this file?  We should probably put a way to override the services to build (an env var or something).

Reply:
Oops, thought I'd reverted these changes